DataDog/documentation
# update ootb rules filtering - [WEB-4062]

### What does this PR do? What is the motivation?

1. updates security rules logic so a rule cannot have two of these rule categories: 
   - `csm-misconfigurations-cloud` , `csm-security-issues` or `csm-identity-risks` 

2. adds source comment to security default rule `.md` files

https://datadoghq.atlassian.net/browse/WEB-4062
### Merge instructions


- [x] Please merge after CorpWeb reviews

preview -> https://docs-staging.datadoghq.com/stefon.simmons/exclude-csm-misconfig-cloud-rule/security/default_rules/
- copy the “**AWS IAM role has a large permissions gap**” rule in the search and ensure it shows for `csm identity risks` rule filtering only.

- You may also check that the `NodeList` for these three queries are empty when you run them in the browser dev console:
  - `document.querySelectorAll('.cat-csm-misconfigurations-cloud.cat-csm-identity-risks')`
  - `document.querySelectorAll('.cat-csm-misconfigurations-cloud.cat-csm-security-issues')`
  - `document.querySelectorAll('.cat-csm-security-issues.cat-csm-identity-risks')`

to test the source comment update:
- clone repo locally
- checkout this branch (stefon.simmons/exclude-csm-misconfig-cloud-rule)
- run `make start`
- check that `content/en/security/default_rules/` files have a comment above the markdown under the frontmatter.
i.e.` `
### Additional notes
